src/main/java/org/jabref/migrations/FileLinksUpgradeWarning.java

```diff
@@ -248,17 +248,13 @@ private void makeChanges(BasePanel panel, ParserResult pr, boolean upgradePrefs,
     }
 
     private boolean showsFileInGenFields() {
-        boolean found = false;
-        EntryEditorTabList tabList = Globals.prefs.getEntryEditorTabList();
-        outer: for (int i = 0; i < tabList.getTabCount(); i++) {
-            List<String> fields = tabList.getTabFields(i);
-            for (String field : fields) {
+        for (Map.Entry<String, List<String>> tab : Globals.prefs.getEntryEditorTabList().entrySet()) {
+            for (String field : tab.getValue()) {
```

Comment:
tab.stream.anymatch( entry -> entry.getKey.equals(FieldName.File)

Reply:
Not sure that this is more readable, especially since the if statement still has to be there.